src/WebJobs.Script/RuntimeInformation/SystemRuntimeInformation.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public class SystemRuntimeInformation : ISystemRuntimeInformation
+    {
+        public Architecture GetOSArchitecture()
+        {
+            return RuntimeInformation.OSArchitecture;
+        }
+
+        public OSPlatform GetOSPlatform()
+        {
+            // Default to Linux
+            OSPlatform os = OSPlatform.Linux;
```

Comment:
instead of defaulting to Linux, return `System.Runtime.InteropServices.RuntimeInformation.OSDescription;`

Reply:
I don't think that's what we would want in this case. `OSDescription` returns the full description of the OS in string form. For example, on my system it returns `Microsoft Windows 10.0.18362`.